Xplat/src/main/java/vazkii/botania/common/item/AstrolabeItem.java

```diff
@@ -254,7 +267,7 @@ public void displayRemainderCounter(Player player, ItemStack stack) {
 
 	private boolean setBlock(ItemStack stack, BlockState state) {
 		if (!state.isAir()) {
-			ItemNBTHelper.setCompound(stack, TAG_BLOCKSTATE, NbtUtils.writeBlockState(state));
+			ItemNBTHelper.setCompound(stack, TAG_BLOCKSTATE, NbtUtils.writeBlockState(state.getBlock().defaultBlockState()));
```

Comment:
maybe add a comment saying we use the blockstate util methods instead of storing the block id directly for legacy/backward compat, might be confusing to future readers